### PR TITLE
Milestone1

### PR DESCRIPTION
**Pull request by fortuneonyeka**
Project set up

### CODE REVIEW

**Review by henatan99 — CHANGES_REQUESTED** [review]
# Changes requested :recycle:
Hi there, nice effort so far. There are some changes you need to make before the project gets approved. 

## To Highligt 
- Correct project structure and Git/Github flow
- Correct linter setup
## To Improve
- Pull Request should have a meaningful title and a brief description [PathWright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
- #!/user/bin/env ruby should be the top line in your main.rb file 
-  `main.rb` should be an [executable file](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/)
- Your commit messages should be descriptive of the changes made to the code 

### Kindly make the requested changes and submit for another review. 

If you have any enquiry, follow the below links and find me,
[LinkedIn](https://www.linkedin.com/in/henok-mekonnen/) | [Twitter](https://twitter.com/henatan99) | Henok Mekonnen on Slack 

## HAppy Coding :computer:

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
# Status:  ♻️ Changes Required ♻️
 Hi @fortuneonyeka and @wajahat414 

### [Required]

You've worked on the previous review but still I can't able to run main file without ruby. To make it you have to put main file inside `bin` folder.


### [Optional]

- [x] I will recommend making main file as `main.rb`.
- [x] There is a typo with `Helloo`. It should be `Hello`.

Remember, This is optional and nothing to do with the project approval.

## Please make the required changes and submit another review request when you are done.

(Feel free to contact me on slack by the handle of  **deep** if you have any additional questions).
Happy Coding
[Dipesh Kumar](https://deeptwist.tk) | [Twitter](https://twitter.com/97deepeshkumar)

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @fortuneonyeka  &  @wajahat414

Good job on implementing the suggested changes. **You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Code review by **Dipesh Kumar**
(Feel free to contact me on slack by the handle of  **deep** if you have any additional questions).

**Pull request merged**
